migrations/migrations/006-form-users.js

```diff
@@ -0,0 +1,13 @@
+const createFormUsersTable = `
+  CREATE TABLE form_users (
+    id SERIAL PRIMARY KEY,
+    template_id INT NOT NULL REFERENCES templates (id),
+    email VARCHAR(80) NOT NULL,
+    temporary_token TEXT,
+    active BOOLEAN NOT NULL DEFAULT TRUE,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    UNIQUE(template_id,active)
```

Comment:
you probably mean template_id and email here right ?

Reply:
lets also use the `CONSTRAINT constraint_name UNIQUE (uc_col1, uc_col2, ... uc_col_n)` format as it makes it easier for down the line migrations

Reply:
CONSTRAINT template_id_email_unique UNIQUE (template_id,email)